modules/spring-boot-autoconfigure/examples/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+ignite:
+  igniteInstanceName: properties-instance-name
+  communicationSpi:
+    localPort: 5555
+  dataStorageConfiguration:
+    defaultDataRegionConfiguration:
+      initialSize: 10485760 #10MB
+    dataRegionConfigurations[0]:
+      name: my-dataregion
+      initialSize: 104857600 #100MB
+  cacheConfiguration[0]:
```

Comment:
Do you check that yml list doesn't work with array?

```
 dataRegionConfigurations:
     - 
      name: my-dataregion
      initialSize: 104857600 #100MB
     -
      name: my-dataregion2
      initialSize: 2342423
...
```

Reply:
Yes it works. Example updated. Thanks.